webpack.config.js

```diff
@@ -61,68 +61,72 @@ const sharedCSSPlugins = [
   new RemoveEmptyScriptsPlugin(),
 ];
 
-const sharedWebpackManifestPlugins = [
-  new WebpackManifestPlugin({
-    // Replace each entry with a prefix of a subdirectory.
-    // NOTE: This could be removed if we update the HTML to use the non-prefixed
-    //       paths.
-    map: (file) => {
-      // if the filename matches .js or .js.map, add js/ prefix if not already present
-      if (file.name.match(/\.js(\.map)?$/)) {
-        if (!file.name.startsWith("js/")) {
-          file.name = `js/${file.name}`; // eslint-disable-line no-param-reassign
-        }
+const sharedWebpackManifestData = {};
+const sharedWebpackManifestMap =
+  // Replace each entry with a prefix of a subdirectory.
+  // NOTE: This could be removed if we update the HTML to use the non-prefixed
+  //       paths.
+  (file) => {
+    // if the filename matches .js or .js.map, add js/ prefix if not already present
+    if (file.name.match(/\.js(\.map)?$/)) {
+      if (!file.name.startsWith("js/")) {
+        file.name = `js/${file.name}`; // eslint-disable-line no-param-reassign
       }
-      // if the filename matches .css or .css.map, add a prefix of css/
-      if (file.name.match(/\.css(\.map)?$/)) {
-        file.name = `css/${file.name}`; // eslint-disable-line no-param-reassign
-      }
-      return file;
-    },
-    // Refs: https://github.com/shellscape/webpack-manifest-plugin/issues/229#issuecomment-737617994
-    publicPath: "",
-  }),
-];
+    }
+    // if the filename matches .css or .css.map, add a prefix of css/
+    if (file.name.match(/\.css(\.map)?$/)) {
+      file.name = `css/${file.name}`; // eslint-disable-line no-param-reassign
+    }
+    return file;
+  };
+// Refs: https://github.com/shellscape/webpack-manifest-plugin/issues/229#issuecomment-737617994
+// publicPath: "",
```

Comment:
question: why is this previously-set value commented out? If it's no longer valid, let's remove it, along with the comment above as to why it's there, or move it to where it's now being set, or extract it to a constant that can be easier commented and reused.

Reply:
addressed in 0f13ec1e6da34b26e9bc7a97c9598aff2c9adae3